docs/reference/ois/next/processing.md

```diff
@@ -339,7 +340,7 @@ endpoints: [
 ]
 ```
 
-## Example #3
+## Post-processing v2 timestamp example
```

Comment:
What about replacing the titles of the other examples?

Reply:
Good idea